src/components/folio-number-input/FolioNumberInput.vue

```diff
@@ -97,7 +110,7 @@ export default class FolioNumberInput extends Vue {
    * @returns True if form is valid and False if not
    */
   public validateFolioNumber (): boolean {
-    return this.$refs.folioForm.validate()
+    return this.$refs.folioForm?.validate()
```

Comment:
Was throwing a console error before.

Reply:
Yes, this happens when this code runs before the component is rendered (ie, folioForm doesn't exist yet).

This is a good solution if validate() doesn't need to be called initially...?